### PR TITLE
Fix the buttons on the model switcher

### LINKED ISSUE #1490: model switcher UX broken on GUI in Juju
The functionality is ok. The selector is broken.
broken-model-switcher

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes #1490.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- change this line to `true` https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/extensions/app-renderer-extension.js#L58
- load the GUI
- open the model switcher
- check that the buttons are aligned correctly

File: jujugui/static/gui/src/app/components/charmbrowser/charmbrowser.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ YUI.add('charmbrowser-component', function() {
       importBundleYAML: React.PropTypes.func.isRequired,
       makeEntityModel: React.PropTypes.func.isRequired,
       renderMarkdown: React.PropTypes.func.isRequired,
-      series: React.PropTypes.array.isRequired,
+      series: React.PropTypes.object.isRequired,
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/charmbrowser/charmbrowser.js]: Drive-by fix for a prop type error

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4780/
Test PASSed.

File: jujugui/static/gui/src/app/components/charmbrowser/charmbrowser.js (hunk 1 above)
Comment (hatched) [inline jujugui/static/gui/src/app/components/charmbrowser/charmbrowser.js]: Ahh thanks for this one!

**Comment by hatched** [comment]
:+1: :ok:
Thanks!
:shipit: shipping as trivial

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**